src/dictionaries/proper_nouns

```diff
@@ -0,0 +1,41 @@
+Aldergrove
+Boolean
+Booleans
```

Comment:
So plurals aren't handled automatically?

Reply:
No, and not surprising given the irregularity of English plurals.